rpm/connman-qt5.spec

```diff
@@ -53,8 +51,11 @@ applications using libconnman-qt
 %qtc_make %{?_smp_mflags}
 
 %install
-rm -rf %{buildroot}
 %qmake5_install
+# MeeGo.Connman legacy import
+mkdir -p %{buildroot}%{_libdir}/qt5/qml/MeeGo/Connman
+ln -sf %{_libdir}/qt5/qml/Connman/libConnmanQtDeclarative.so %{buildroot}%{_libdir}/qt5/qml/MeeGo/Connman/
```

Comment:
This might be better if it was relative. This seems to annoy our SDK:
```
Resetting snapshot 'SailfishOS-devel-armv7hl.default' of 'SailfishOS-devel-armv7hl' target...
cp: not writing through dangling symlink '/srv/mer/targets/SailfishOS-devel-armv7hl.default/usr/lib/qt5/qml/MeeGo/Connman/libConnmanQtDeclarative.so'
Removing destination and trying again...
```

Reply:
Copied the setup from some existing nemo module and it was working anyhow, but then again the relative version could be even a bit simpler so let's just do that.